torch/csrc/autograd/enum_tags.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#include <torch/csrc/python_headers.h>
+
+namespace torch {
+    namespace autograd {
+    void initEnumTags(PyObject* module);
```

Comment:
doesn't feel good putting in `torch::autograd`. @albanD where should this go?

Reply:
The directory structure in torch/csrc is irredeemably hopeless. Just do something that locally makes sense.